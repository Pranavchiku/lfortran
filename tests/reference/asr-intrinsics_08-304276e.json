{
    "basename": "asr-intrinsics_08-304276e",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_08.f90",
    "infile_hash": "cf340bcd7898a8a082647dcd2f5bdd123faa7de9b36f4ac6f31ab4c2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_08-304276e.stdout",
    "stdout_hash": "eafa88853416d875eedd0637dffa90fc87178b0443cdd072f3a23ff7",
    "stderr": "asr-intrinsics_08-304276e.stderr",
    "stderr_hash": "a22d60dd805afc4652b401a081a92e0cd602bffd86c8e1b3d8df5ab7",
    "returncode": 0
}