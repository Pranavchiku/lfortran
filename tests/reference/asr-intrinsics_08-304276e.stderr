warning: Implied save feature
 --> tests/../integration_tests/intrinsics_08.f90:2:3
  |
2 |   real, parameter :: x=tiny(1.0)
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/intrinsics_08.f90:3:3
  |
3 |   real (kind = 8) :: b
  |   ^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/intrinsics_08.f90:4:3
  |
4 |   real (kind = 8), parameter :: y=tiny(b)
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
